src/layouts/footer.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>FOOTER</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.6.0/css/all.min.css">
+    <link href="../../assets/css/output.css" rel="stylesheet">
+</head>
+<body>
+    <footer id="footer" class="bg-neutral-950 text-gray-400 font-openSans">
+        <div class="container mx-auto max-w-7xl flex flex-col lg:flex-row flex-wrap pt-12 pb-10 ">
+            <div class="lg:w-1/3 p-4 pr-24">
+                <div class="text-white text-xl mb-5" data-i18n="footer.contact_us">
+                    LIÊN HỆ VỚI CHÚNG TÔI
+                </div>
+                <ul class="text-sm space-y-2">
+                    <li>
+                        <i class="fa-solid fa-location-dot text-primary-color"></i>
+                        <span data-i18n="footer.address">Tầng 4, Tòa nhà Haux Group. 56 442 Đội Cấn, P. Cống Vị, Q. Ba Đình, Hà Nội</span>
+                    </li>
+                    <li>
+                        <i class="fa-solid fa-phone text-primary-color"></i>
+                        <span>(04) 6674 2332 - </span>
+                        <i class="fa-solid fa-phone-volume text-primary-color"></i>
+                        <span>(04) 3716 8904</span>
+                    </li>
+                    <li>
+                        <i class="fa-solid fa-phone text-primary-color"></i>
+                        <span>(08) 66809685&nbsp;&nbsp;&nbsp;</span>
+                        <i class="fa-solid fa-envelope text-primary-color"></i>
+                        <span class="text-primary-color italic">
+                            Support@bizweb.vn
+                        </span>
+                        
+                    </li>
+                </ul>
+            </div>
+            <div class="lg:w-1/6 p-4">
+                <div class="text-white text-xl mb-5" data-i18n="footer.shipping">
+                    CHUYỂN HÀNG
+                </div>
+                <ul class="text-sm space-y-2">
+                    <li>
+                        <a href="" class="transition hover:text-primary-color hover:transform hover:scale-105 hover:translate-x-2 
+                        duration-300 inline-block" data-i18n="footer.online_shopping">
+                            Mua sắm trực tuyến
+                        </a>
+                    </li>
+                    <li>
+                        <a href="" class="transition hover:text-primary-color hover:transform hover:scale-105 hover:translate-x-2 
+                        duration-300 inline-block" data-i18n="footer.from_us_to_you">
+                            Đến từ chúng tôi gửi đến
+                        </a>
```

Comment:
```suggestion
                        <a href="" class="transition hover:text-primary-color hover:transform hover:scale-105 hover:translate-x-2 
                        duration-300 inline-block" data-i18n="footer.from_us_to_you"></a>
```
Mình có html rồi thì sẽ k dùng text nữa nha, e xóa nhé, tương tự những chỗ khác